contrib/debian/control

```diff
@@ -13,9 +13,9 @@ Vcs-Browser: https://github.com/zcash/zcash
 Package: zcash
 Architecture: amd64
 Depends: ${shlibs:Depends}
-Description: HTTPS for money.
+Description: Zcash libraries and tools
  Based on Bitcoin's code, it intends to offer a far higher standard
  of privacy and anonymity through a sophisticiated zero-knowledge
  proving scheme which preserves confidentiality of transaction metadata.
  This package provides the daemon, zcashd, and the CLI tool,
- zcash-cli, to interact with the daemon.
+ zcash-cli, to interact with the daemon. Think of it as HTTPS for money.
```

Comment:
Nit: I would put the new text before "This package provides [...]".

Reply:
:+1: